packages/editor-ui/src/mixins/pushConnection.ts

```diff
@@ -295,7 +295,7 @@ export const pushConnection = mixins(
 						for (const key of Object.keys(activeRunData)) {
 							if (
 								pushData.data.data.resultData.runData[key]?.[0]?.data?.main?.[0]?.[0]?.json
-									.isArtificalRecoveredEventItem === true &&
+									?.isArtificalRecoveredEventItem === true &&
```

Comment:
while we are at it, can you please rename all typos `Artifical` to `Artificial` across the repo 🙏🏽

Reply:
Good catch! Done.